<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/iLogos/logo_novafct.png" width="200">

# Departamento de Engenharia Mecânica e Industrial
## Mecânica Aplicada I 

## EQUILÍBRIO DA PARTÍCULA NO ESPAÇO

### Problema 7 <a name="P12"></a>

Duas roldanas de 150 mm de diâmetro estão montadas sobre um veio $AD$, como se representa na figura. As correias em $B$ e $C$ estão contidas em planos verticais paralelos ao plano $yz$. As chumaceiras em $A$ e $D$ impedem movimentos de translação transversais ao veio e a chumaceira $A$ impede também movimentos axiais de translação.

[a)](#A) Diga o que entende por binário? Qual a importante propriedade que este vetor apresenta?

[b)](#B) Reduza o sistema de forças em cada roldana a um sistema força-binário equivalente no centro da roldana.

[c)](#C) Desenhe o diagrama de corpo livre do conjunto da figura (sugestão: aproveite as simplificações da alínea anterior).

[d)](#D) Calcule as reações nos apoios $A$ e $D$.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/MAI_Ch04_P7.png"
width="500">

### Resolução

#### a) Diga o que entende por binário? Qual a importante propriedade que este vetor apresenta?. <a id="A"></a> ([_enunciatum_](#P9))

O momento binário representa a ação de rotação gerada sobre um corpo rígido por um par de forças de igual intensidade e sentidos opostos. O vetor momento binário é perpendicular ao plano que contém as duas forças.

O momento binário é uma quantidade fundamental na análise do equilíbrio de corpos rígidos pois permite a simplificação do sistema de forças e momentos atuantes sobre o mesmo. É possível reduzir um sistema de forças e momentos aplicados a um corpo rígido a um sistema equivalente consistindo apenas uma força resultante e um momento binário. A ideia chave desta transferência é o binário.

#### b) Reduza o sistema de forças em cada roldana a um sistema força-binário equivalente no centro da roldana. <a id="B"></a> ([_enunciatum_](#P9))

Embora se possa mover uma força $\vec{F}$ ao longo da sua linha de ação, pelo princípio da transmissibilidade, sem alterar o equilíbrio estático de um objeto, não se pode mover arbitrariamente o ponto de aplicação da força $\vec{F}$ (um novo ponto que não pertença à sua linha de ação), sem modificar a ação de $\vec{F}$ sobre o corpo.

Contudo, qualquer força $\vec{F}$ atuando num corpo rígido pode ser movida para um ponto arbitrário $O$, desde que seja acrescentado um binário cujo momento é igual ao momento gerado pela ação de rotação de $\vec{F}$ em relação ao ponto $O$. A combinação obtida designa se **sistema força-binário**.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/MAI_Ch04_P7a.jpg"
width="500">

Generalizando a condição anterior, qualquer sistema de forças, $\vec{F}_\textrm{1},\vec{F}_\textrm{2},\ldots,\vec{F}_\textrm{n}$ pode ser substituído por um __sistema força-binário equivalente__ aplicado num dado ponto O, constituído pelos vetores $\vec{R}$ (resultante das forças) e $\vec{M}_O^R$ (resultante dos momentos das forças).

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/SisEqForBin.jpg"
width="500">

O sistema força-binário (força e momento resultante) é obtido pelas equações:

\begin{equation*}
\begin{cases}
\vec{R} &=& \sum\limits_{i} \vec{F}_\textrm{i} \\
\vec{M}_\textrm{O}^R &=& \sum \vec{M}_\textrm{O}
= \sum\limits_{i} (\vec{r}_\textrm{Oi}\times\vec{F}_\textrm{i}) \\
\end{cases}
\end{equation*}

**Roldana B**

Comece-se por considerar o diagrama de corpo livre da Roldana B (por conveniência numa visão 3D e 2D).

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/MAI_Ch04_P7b.jpg"
width="500">

Nos esquemas seguintes ilustra-se:

1. A aplicação do princípio de equivalência para a definição do sistema força-binário para o exemplo da força $\vec{F}_1$ (embora não ilustrado, o mesmo princípio se aplica para a força $\vec{F}_2$).
2. DCL da roldana B, representando todas as forças e momentos aplicados bem como as suas resultantes.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/MAI_Ch04_P7b2.jpg"
width="500">

* Cálculo do sistema força-binário equivalente

A resultante das forças aplicadas na roldana B:

\begin{equation*}
\vec{F}_\textrm{B}^\textrm{R} =
\vec{F}_\textrm{1} + \vec{F}_\textrm{2}
\end{equation*}

In [1]:
import numpy as np
import sympy as sy
from sympy.solvers import solve
from sympy.physics.mechanics import ReferenceFrame, Vector, dot, cross

F1 = 145 # unit: N
F2 = 215 # unit: N
angB = np.deg2rad(20) # unit: radianos

S = ReferenceFrame('S') # ||x|
vF1 = -F1*np.cos(angB)*S.y + F1*np.sin(angB)*S.z
print('F1 : ',vF1)
vF2 = -F2*S.y
print('F2 : ',vF2)
VFBR = vF1 + vF2
print('FBres : ',VFBR)

F1 :  - 136.255430013957*S.y + 49.592920782222*S.z
F2 :  - 215*S.y
FBres :  - 351.255430013957*S.y + 49.592920782222*S.z


onde,

\begin{equation*}
\begin{cases}
\vec{F}_\textrm{1} &= - 136.3\,\vec{\jmath} + 49.6\,\vec{k}\\
\vec{F}_\textrm{2} &= - 215\,\vec{\jmath}\\
\vec{F}_\textrm{B}^\textrm{R} &= - 351.3\,\vec{\jmath} + 49.6\,\vec{k}
\end{cases}
\end{equation*}

A resultante dos momentos das forças em relação ao ponto B:

\begin{equation*}
\vec{M}_\textrm{B}^\textrm{R}
= \sum\limits_{i} (\vec{r}_\textrm{Oi}\times\vec{F}_\textrm{i})
= (\vec{r}_\textrm{1}\times\vec{F}_\textrm{1}) +
(\vec{r}_\textrm{2}\times\vec{F}_\textrm{2})
\end{equation*}

In [2]:
raio = 150/2/1e3 # unit: mm2m

r1 = raio*np.sin(angB)*S.y + raio*np.cos(angB)*S.z
print('r1 : ',r1)
r2 = -raio*S.z
print('r2 : ',r2)

r1 :  0.0256515107494252*S.y + 0.0704769465589431*S.z
r2 :  - 0.075*S.z


com,

\begin{equation*}
\begin{cases}
\vec{r}_\textrm{1} &= - 0.0257\,\vec{\jmath} + 0.0705\,\vec{k}\\
\vec{r}_\textrm{2} &= - 0.075\,\vec{\jmath}
\end{cases}
\end{equation*}

Os produtos vetoriais podem ser determinados pelo cálculo dos seguintes determinantes:

\begin{equation*}
\bullet\quad\vec{r}_\textrm{1}\times\vec{F}_\textrm{1} =
\begin{vmatrix}
\vec{\imath} & \vec{\jmath} & \vec{k} \\
0 & r_\textrm{1y} & r_\textrm{1z} \\
0 & F_\textrm{1y} & F_\textrm{1z}
\end{vmatrix} =
\begin{vmatrix}
r_\textrm{1y} & r_\textrm{1z} \\
F_\textrm{1y} & F_\textrm{1z}
\end{vmatrix}\vec{\imath}
\end{equation*}

\begin{equation*}
\qquad\qquad~~~ = (r_\textrm{1y}F_\textrm{1z} - r_\textrm{1z}F_\textrm{1y})
\vec{\imath} = (10.88)\vec{\imath}
\end{equation*}

In [3]:
CP1 = cross(r1,vF1); print('Cross 1: ',CP1)

Cross 1:  10.875*S.x


\begin{equation*}
\bullet\quad\vec{r}_\textrm{2}\times\vec{F}_\textrm{2} =
\begin{vmatrix}
\vec{\imath} & \vec{\jmath} & \vec{k} \\
0 & 0 & -r_\textrm{2z} \\
0 & -F_\textrm{2y} & 0
\end{vmatrix} =
\begin{vmatrix}
0 & -r_\textrm{2z} \\
F_\textrm{2y} & 0
\end{vmatrix}\vec{\imath}
\end{equation*}

\begin{equation*}
\qquad\qquad~~~ = (r_\textrm{2z}F_\textrm{2y})
\vec{\imath} = (- 16.13)\vec{\imath}
\end{equation*}

In [4]:
CP2 = cross(r2,vF2); print('Cross 2: ',CP2)

Cross 2:  - 16.125*S.x


Resulta finalmente:

\begin{equation*}
\vec{M}_\textrm{B}^\textrm{R}
= (\vec{r}_\textrm{1}\times\vec{F}_\textrm{1}) +
(\vec{r}_\textrm{2}\times\vec{F}_\textrm{2}) =
(10.88)\vec{\imath} +
(- 16.13)\vec{\imath} =
- 5.25 \vec{\imath}
\end{equation*}

In [5]:
MBR  = CP1 + CP2
print('MBR: ',MBR)

MBR:  - 5.25*S.x


**Roldana C**

Considere-se agora o diagrama de corpo livre da Roldana C (por conveniência numa visão 3D e 2D).

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/MAI_Ch04_P7c.jpg"
width="500">

Calculando a resultante das forças aplicas na roldana C:

\begin{equation*}
\vec{F}_\textrm{C}^\textrm{R} =
\vec{F}_\textrm{3} + \vec{F}_\textrm{4}
\end{equation*}

In [6]:
F3 = 155 # unit: N
F4 = 225 # unit: N
angC = np.deg2rad(10) # unit: radianos

vF3 = -F3*np.sin(angC)*S.y - F3*np.cos(angC)*S.z
print('F3 : ',vF3)
vF4 = -F4*np.sin(angC)*S.y - F4*np.cos(angC)*S.z
print('F4 : ',vF4)
VFCR = vF3 + vF4
print('FCres : ',VFCR)

F3 :  - 26.9154675383742*S.y - 152.645201716892*S.z
F4 :  - 39.0708399750593*S.y - 221.581744427747*S.z
FCres :  - 65.9863075134335*S.y - 374.226946144639*S.z


onde,

\begin{equation*}
\begin{cases}
\vec{F}_\textrm{3} &= - 26.9\,\vec{\jmath} - 152.6\,\vec{k}\\
\vec{F}_\textrm{4} &= - 39.1\,\vec{\jmath} - 221.6\,\vec{k}\\
\vec{F}_\textrm{C}^\textrm{R} &= - 65.99\,\vec{\jmath} -374.23\,\vec{k}
\end{cases}
\end{equation*}

A resultante dos momentos das forças em relação ao ponto C:

\begin{equation*}
\vec{M}_\textrm{C}^\textrm{R}
= \sum\limits_{i} (\vec{r}_\textrm{Oi}\times\vec{F}_\textrm{i})
= (\vec{r}_\textrm{3}\times\vec{F}_\textrm{3}) +
(\vec{r}_\textrm{4}\times\vec{F}_\textrm{4})
\end{equation*}

In [7]:
r3 = raio*np.cos(angC)*S.y - raio*np.sin(angC)*S.z
print('r3 : ',r3)
r4 = -raio*np.cos(angC)*S.y + raio*np.sin(angC)*S.z
print('r4 : ',r4)

r3 :  0.0738605814759156*S.y - 0.0130236133250198*S.z
r4 :  - 0.0738605814759156*S.y + 0.0130236133250198*S.z


com,

\begin{equation*}
\begin{cases}
\vec{r}_\textrm{3} &= 0.0739\,\vec{\jmath} - 0.013\,\vec{k}\\
\vec{r}_\textrm{4} &= - 0.0739\,\vec{\jmath}
\end{cases}
\end{equation*}

Os produtos vetoriais podem ser determinados pelo cálculo dos seguintes determinantes:

\begin{equation*}
\bullet\quad\vec{r}_\textrm{3}\times\vec{F}_\textrm{3} =
\begin{vmatrix}
\vec{\imath} & \vec{\jmath} & \vec{k} \\
0 & r_\textrm{3y} & -r_\textrm{3z} \\
0 & -F_\textrm{3y} & -F_\textrm{3z}
\end{vmatrix} =
\begin{vmatrix}
r_\textrm{3y} & -r_\textrm{3z} \\
-F_\textrm{3y} & -F_\textrm{3z}
\end{vmatrix}\vec{\imath}
\end{equation*}

\begin{equation*}
\qquad\qquad~~~ = (-r_\textrm{3y}F_\textrm{3z} - r_\textrm{3z}F_\textrm{3y})
\vec{\imath} = (-11.63)\vec{\imath}
\end{equation*}

In [8]:
CP3 = cross(r3,vF3); print('Cross 3: ',CP3)

Cross 3:  - 11.625*S.x


\begin{equation*}
\bullet\quad\vec{r}_\textrm{4}\times\vec{F}_\textrm{4} =
\begin{vmatrix}
\vec{\imath} & \vec{\jmath} & \vec{k} \\
0 & -r_\textrm{4y} & -r_\textrm{4z} \\
0 & -F_\textrm{4y} & -F_\textrm{4z}
\end{vmatrix} =
\begin{vmatrix}
-r_\textrm{1y} & -r_\textrm{1z} \\
-F_\textrm{1y} & -F_\textrm{1z}
\end{vmatrix}\vec{\imath} =
(16.88)\vec{\imath}
\end{equation*}

In [9]:
CP4 = cross(r4,vF4); print('Cross 4: ',CP4)

Cross 4:  16.875*S.x


\begin{equation*}
\vec{M}_\textrm{C}^\textrm{R}
= (\vec{r}_\textrm{3}\times\vec{F}_\textrm{3}) +
(\vec{r}_\textrm{4}\times\vec{F}_\textrm{4}) =
(-11.63)\vec{\imath} +
(16.88)\vec{\imath} =
5.25 \vec{\imath}
\end{equation*}

In [10]:
MCR  = CP3 + CP4
print('MCR: ',MCR)

MCR:  5.25*S.x


#### c) Desenhe o diagrama de corpo livre do conjunto da figura (sugestão: aproveite as simplificações da alínea anterior) <a id="C"></a> ([_enunciatum_](#P9))

Quando as forças que atuam sobre um corpo podem ser reduzidas, num ponto arbitrário O, a um sistema força-binário constituído por uma força nula e um binário nulo, diz-se que o corpo está em equilíbrio estático.

Algebricamente, as condições necessárias e suficientes para garantir o equilíbrio estático expressa-se por:

\begin{equation*}
\bullet\quad\sum \vec{F} = \vec{0}~;
\quad\sum \vec{M}_\textrm{O} =
\sum (\vec{r}_\textrm{Oi}\times\vec{F}_\textrm{i}) = \vec{0}
\end{equation*}

Decompondo a força e o momento nas suas componentes cartesianas, estas duas equações vetoriais conduzem às seguintes seis equações escalares:

\begin{equation*}
\bullet\quad\sum \vec{F} = \vec{0}~\Leftrightarrow~
\begin{cases}
\rightarrow(+)&~x~:&\sum F_x = 0 \\
\uparrow(+)&~y~:&\sum F_y = 0 \\
\swarrow(+)&~z~:&\sum F_z = 0 \\
\end{cases}
\end{equation*}

e

\begin{equation*}
\bullet\quad\sum \vec{M}_A = \vec{0}~\Leftrightarrow~
\begin{cases}
\vec{k}\times\vec{\jmath} = \vec{\imath}~(+)&:&\sum M_{Ax} = 0 \\
\vec{k}\times\vec{\imath} = \vec{\jmath}~(+)&:&\sum M_{Ay} = 0 \\
\vec{\imath}\times\vec{\jmath} = \vec{k}~(+)&:&\sum M_{Az} = 0 \\
\end{cases}
\end{equation*}

Para escrever as equações de equilíbrio de um corpo rígido é essencial identificar todas as forças aplicadas no corpo. Essas forças devem constar do diagrama de corpo livre.

O diagrama de corpo livre é um desenho simplificado do corpo ou da parte do corpo que se pretende estudar, desligado dos apoios e separado de todos os outros corpos. Para a construção do diagrama de corpo livre considere-se:

- Identificar todas as forças exteriores aplicadas sobre o corpo. Essas forças representam a ação dos outros corpos, do peso próprio (aplicado no centro de gravidade) e dos apoios e estão aplicadas no ponto de contacto com o corpo.

- As intensidades, direções e sentidos das forças exteriores conhecidas deverão estar claramente representadas.

- As forças exteriores desconhecidas consistem geralmente em
reações nos apoios.

- O diagrama de corpo livre deve incluir as dimensões necessárias à
resolução do problema

Identificação das forças externas aplicadas ao veio AD:

* DCL 1:

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/MAI_Ch04_P7d.jpg"
width="500">

* DCL 2 (incorporando sistema força-binário):

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/MAI_Ch04_P7e.jpg"
width="500">


#### d) Calcule as reações nos apoios A e D. <a id="D"></a> ([_enunciatum_](#P9))

**Equilíbrio de Forças**

\begin{equation*}
\begin{cases}
\sum F_x = 0 \\
\sum F_y = 0 \\
\sum F_z = 0 \\
\end{cases}~\Leftrightarrow~
\begin{cases}
R_\textrm{Ax} = 0 \\
R_\textrm{Ay} - F_\textrm{By}^\textrm{R} - F_\textrm{Cy}^\textrm{R} +
R_\textrm{Dy} = 0 \\
R_\textrm{Az} - F_\textrm{Bz}^\textrm{R} - F_\textrm{Cz}^\textrm{R} +
R_\textrm{Dz} = 0 \\
\end{cases}
\end{equation*}

In [11]:
rax, ray, raz, rdy, rdz = sy.symbols('rax ray raz rdy rdz')

EFX = rax
EFY = ray + VFBR.args[0][0].args[2][1] + VFCR.args[0][0].args[2][1] + rdy
EFZ = raz + VFBR.args[0][0].args[2][2] + VFCR.args[0][0].args[2][2] + rdz

**Equilíbrio de Momentos**

\begin{equation*}
\begin{aligned}
\sum \vec{M}_\textrm{A} &=
\sum (\vec{r}_\textrm{Ai}\times\vec{F}_\textrm{i}) \\
&= (\vec{r}_\textrm{AB}\times\vec{F}_\textrm{B}^\textrm{R})
+ (\vec{r}_\textrm{AC}\times\vec{F}_\textrm{C}^\textrm{R})
+ (\vec{r}_\textrm{AD}\times\vec{R}_\textrm{D})
+ \vec{M}_\textrm{B}^\textrm{R} + \vec{M}_\textrm{C}^\textrm{R} = \vec{0}
\end{aligned}
\end{equation*}

com,

\begin{equation*}
\begin{cases}
\vec{r}_\textrm{AB} &=& 0.225\,\vec{\imath} \\
\vec{r}_\textrm{AC} &=& 0.450\,\vec{\imath} \\
\vec{r}_\textrm{AD} &=& 0.630\,\vec{\imath} \\
\vec{F}_\textrm{B}^\textrm{R} &=&
-351.3\,\vec{\jmath} + 49.6\,\vec{k} \\
\vec{F}_\textrm{C}^\textrm{R} &=&
-65.99\,\vec{\jmath} - 374.23\,\vec{k} \\
\vec{R}_\textrm{D} &=& R_\textrm{Dy}\,\vec{\jmath} + R_\textrm{Dz}\,
\vec{k}\\
\vec{M}_\textrm{B}^\textrm{R} &=& -5.25 \,\vec{\imath} \\
\vec{M}_\textrm{C}^\textrm{R} &=& 5.25\,\vec{\imath}
\end{cases}
\end{equation*}

Resultando:

\begin{equation*}
\begin{cases}
\sum M_\textrm{Ax} = 0 \\
\sum M_\textrm{Ay} = 0 \\
\sum M_\textrm{Az} = 0 \\
\end{cases}~\Leftrightarrow~
\begin{cases}
-5.25 + 5.25= 0 ~\textrm{(proposição verdadeira)}\\
-11.16 + 168.4 - 0.63\,R_\textrm{Dz} = 0 \\
-76.03 - 29.7 + 0.63\,R_\textrm{Dy} = 0 \\
\end{cases}
\end{equation*}

In [12]:
rAB = 0.225*S.x
CrossI = cross(rAB,VFBR); print('Cross I : ',CrossI)

rAC = 0.45*S.x
CrossII = cross(rAC,VFCR); print('Cross II : ',CrossII)

rAD = 0.630*S.x
vRD = rdy*S.y + rdz*S.z
CrossIII = cross(rAD,vRD); print('Cross III : ',CrossIII,'\n')

EMX = MBR.args[0][0].args[2][0] + MCR.args[0][0].args[2][0]
EMY = CrossI.args[0][0].args[2][1] + \
      CrossII.args[0][0].args[2][1] + \
      CrossIII.args[0][0].args[2][1]
EMZ = CrossI.args[0][0].args[2][2] + \
      CrossII.args[0][0].args[2][2] + \
      CrossIII.args[0][0].args[2][2]

sol = solve({EFX,EFY,EFZ,EMY,EMZ},{rax,ray,raz,rdy,rdz})

RAx = float(sol[rax]); print('RAx = ','{:.1f}'.format(RAx),' N')
RAy = float(sol[ray]); print('RAy = ','{:.1f}'.format(RAy),' N')
RAz = float(sol[raz]); print('RAz = ','{:.1f}'.format(RAz),' N')
RDy = float(sol[rdy]); print('RDy = ','{:.1f}'.format(RDy),' N')
RDz = float(sol[rdz]); print('RDz = ','{:.1f}'.format(RDz),' N')

Cross I :  - 11.1584071759999*S.y - 79.0324717531403*S.z
Cross II :  168.402125765088*S.y - 29.6938383810451*S.z
Cross III :  - 0.63*rdz*S.y + 0.63*rdy*S.z 

RAx =  0.0  N
RAy =  244.7  N
RAz =  75.0  N
RDy =  172.6  N
RDz =  249.6  N


**Resumo**

Da análise do equilíbrio estático foi possível determinar a intensidades de todas as forças identificadas e representadas no DCL:

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/MAI_Ch04_P7f.jpg"
width="500">

ou com base nos sistemas força-binário

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/MAI_Ch04_P7g.png"
width="500">

### Apêndice A

**Categorias de equações de equilíbrio no plano (3D)**

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/GFormulas/EQ3D.jpg"
width="700">


<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/GFormulas/TabReac2.png"
width="700">

### Apêndice B

Genericamente, uma chumaceira pode exercer sobre um veio quer forças de reação quer momentos de reação. De notar que o momento na reação será sempre perpendicular ao eixo de rotação do veio. Contudo, este orgão de máquina é tipicamente projetado para absorver forças radiais e axiais. Consequentemente, o modelo matemático do sistema mecânico despreza as reações de momento na chumaceira, considerando apenas forças de reação.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P7/AppB.png"
width="400">

---

Copyright (c) DEMI - FCT NOVA

Interactive computing by <a href="https://jupyter.org/" target="_blank"> <span
style="color:#333399"> Jupyter Notebook </span> </a> &nbsp;|&nbsp;Coded by <a href = "mailto: jmc.xavier@fct.unl.pt">José Xavier</a>

Licensed under  <a href="http://creativecommons.org/licenses/by-sa/4.0/"
target="_blank"> <span style="color:#333399;font-size: 20px"> CC BY-SA 4.0  </span></a>